fiatconnect-api.md

```diff
@@ -2352,6 +2353,37 @@ The `ifsc` or Indian Financial System Code is an 11-digit alpha-numeric code tha
 The `accountNumber` field is unique between individuals and no two banks or account holders can have the same account number.
 Banks use different starting codes for their branches for differentiation. (In India, bank account numbers usually contain 7 to 21 digits.)
 
+#### 9.3.2.6. `PIXAccount`
+
+`PIXAccount`  egg. [PIX](https://www.bcb.gov.br/estabilidadefinanceira/pix) is a Brazilian instant payment method created by the Central Bank (BC) in which funds are transferred between accounts in a few seconds, at any time or day. Pix transfers can be carried out from a checking account, savings account or prepaid payment account.
```

Comment:
```suggestion
`PIXAccount` is a fiat account type for [PIX](https://www.bcb.gov.br/en/financialstability/pix_en), a Brazilian instant payment method created by the Brazilian Central Bank in which funds are transferred between accounts in a few seconds, at any time or day. Pix transfers can be carried out from a checking account, savings account or prepaid payment account.
```
I rephrased this for clarity, removed a typo (egg. instead of e.g.? but e.g. means 'for example' so it didn't make sense there anyway, I just removed it), and replaced a portuguese informative reference with one in English. The spec is written in English, so all informative references should also be in English, where possible.